src/main/java/org/opensearch/knn/index/query/KNNQueryBuilder.java

```diff
@@ -6,6 +6,7 @@
 package org.opensearch.knn.index.query;
 
 import lombok.extern.log4j.Log4j2;
+import org.opensearch.core.Strings;
```

Comment:
fixed it, tried because of a failing integ tests, it appears that previous package was correct and instead recent core changes are not yet published for intg-test-zip artifact